iiblib/iibclient.py

```diff
@@ -390,7 +398,14 @@ class IIBClient(object):
     """IIB requests wrapper"""
 
     def __init__(
-        self, hostname, retries=3, auth=None, poll_interval=30, ssl_verify=True
+        self,
+        hostname,
+        retries=3,
+        auth=None,
+        poll_interval=30,
+        ssl_verify=True,
+        backoff_factor=2,
+        wait_for_build_timeout=7200,
```

Comment:
Can IIB operations really take as much as 2 hours? If yes, do we have a guarantee that they won't take longer?

Reply:
No, I guess we do not have it guaranteed. I browsed all requests on prod and the longest requests took 0:14:43.